client/landing/stepper/declarative-flow/trial-wooexpress-flow.ts

```diff
@@ -78,12 +79,20 @@ const wooexpress: Flow = {
 			const redirectTarget =
 				`/setup/wooexpress` +
 				( hasFlowParams ? encodeURIComponent( '?' + flowParams.toString() ) : '' );
-			// Early return approach
-			if ( locale || locale === 'en' ) {
-				return `/start/account/user/${ locale }?variationName=${ flowName }&redirect_to=${ redirectTarget }`;
+
+			let startUrl = '/start/account/user';
+
+			if ( locale ) {
+				startUrl += `/${ locale }`;
+			}
+
+			startUrl += `?variationName=${ flowName }&redirect_to=${ redirectTarget }`;
+
+			if ( aff ) {
+				startUrl += `&aff=${ aff }`; // Add 'aff' parameter if it's not empty
```

Comment:
Can we supply the params to the URL object, instead of mutating this string. The order of the params doesn't matter, but it would be easier to maintain if we supply a param name and a value, instead of keeping track of the string and `?` vs `&` I think.